### ISSUE TITLE
Classification task training issue

### ISSUE THREAD

**Issue by JackLiao0824** [ysj9909/SHViT#3]
Hi,
I'm trying to train your SHVIT model from S1 to S4 on the ImageNet-1k classification task, unfortunately, all my attempts have failed.The highest accuracy achieved was 0.15%.Did you make any modifications to the training code?
thanks.

**Comment by ysj9909** [comment]
Based on today's review, there are no issues with the code. Please check if there are any problems with the dataset you used or if you have modified the code.

**Comment by eric900115** [comment]
Same problem here! I am unable to achieve the same accuracy mentioned in the paper when retraining from scratch using the provided README. The accuracy remains around 25% even after training for 120 epochs. Changing the batch size to 2000 did not resolve the issue as well.
